### PR TITLE
Code refactoring

### PR DESCRIPTION
**Pull request by surajdm123**

### CODE REVIEW

File: server/app/controllers/checks_controller.py [hunk 1]
```diff
@@ -153,6 +164,8 @@ def run(check_id):
             raise ValueError(
                 "Check with id {} is not present.".format(check_id))
 
+    
+    # This method returns a list of all the checks based on the course_id.
```
Comment (efg) [inline server/app/controllers/checks_controller.py:168]: Does that mean it returns a list of all checks that pertain to a particular course_id, or that it returns a list of checks, ordered by course_id?

File: server/app/controllers/checks_controller.py [hunk 2]
```diff
@@ -169,6 +182,8 @@ def show_checks(parameters):
                                 'start_date': check.start_date.date()})
         return checks_list
 
+    # This disables the check from triggering MOSS and 
+    # also sets visibility to false in the database.
```
Comment (efg) [inline server/app/controllers/checks_controller.py:186]: If visibility is false, what does that mean?  That MOSS won't be run on it anymore?  That it won't show up in a list?  Please explain.

File: server/app/controllers/checks_controller.py [hunk 3]
```diff
@@ -208,6 +225,7 @@ def disable_check(parameters):
                 db.session.commit()
                 ChecksController.remove_scheduled_job(check_id)
 
+    # This method returns the status of the check.
```
Comment (efg) [inline server/app/controllers/checks_controller.py:228]: This just begs the question; what does status mean?  What are its possible values?

File: server/app/controllers/checks_controller.py (hunk 1 above)
Comment (surajdm123) [reply server/app/controllers/checks_controller.py:168]: That method returns a list of all checks for a particular `course_id`.  I'll rephrase the comment.

File: server/app/controllers/checks_controller.py (hunk 2 above)
Comment (surajdm123) [reply server/app/controllers/checks_controller.py:186]: If the visibility is false, the check will not be displayed in the UI.

File: server/app/controllers/checks_controller.py (hunk 3 above)
Comment (surajdm123) [reply server/app/controllers/checks_controller.py:228]: This method returns `true` if the check is active and returns `false` otherwise.

Comment (surajdm123) [reply server/app/controllers/checks_controller.py:228]: I'll rephrase the comment to include the return types.

**Pull request merged**